services/wallet/service.go

```diff
@@ -149,7 +149,7 @@ type Service struct {
 }
 
 type DAppConfig struct {
-	AllowedOrigin string
+	AllowedOrigin []string
```

Comment:
Re: the comment earlier, would it make sense to change it here?

```suggestion
	AllowedOrigins []string
```